microprofile.ls/com.redhat.microprofile.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-properties.json

```diff
@@ -3,10 +3,10 @@
 		"prefix": "qds",
 		"body": [
 			"quarkus.datasource.url=${1|jdbc:mariadb://localhost:3306/mydb,jdbc:mysql://localhost:3306/test,jdbc:h2:mem:mydb,jdbc:postgresql://localhost/mydb,jdbc:sqlserver://localhost:1433;databaseName=mydb|}",
```

Comment:
looks like it's missing a derby url, like "jdbc:derby: mydb". See https://builds.apache.org/job/Derby-docs/lastSuccessfulBuild/artifact/trunk/out/getstart/rgsquck30197.html

Reply:
Added. I also added the default port for postgresql